dspace-api/src/main/java/org/dspace/access/status/AccessStatusHelper.java

```diff
@@ -27,4 +27,14 @@ public interface AccessStatusHelper {
      */
     public String getAccessStatusFromItem(Context context, Item item, Date threshold)
         throws SQLException;
+
+    /**
+     * Retrieve embargo information for the item
+     *
+     * @param context the DSpace context
+     * @param item the item to check for embargo information
+     * @return an embargo date
+     * @throws SQLException An exception that provides information on a database access error or other errors.
+     */
+    public String getEmbargoFromItem(Context context, Item item) throws SQLException;
```

Comment:
@paulo-graca : I'm personally OK with this approach because of how `DefaultAccessStatusHelper` is currently implemented.

While I *agree* it'd be nice if this AccessStatusHelper (and DefaultAccessStatusHelper) was aligned with the configurations in `access-conditions.xml`, currently it's entirely separate.  It ignores the configurations and applies it's own "access status" labels based entirely on our "State Definitions" at https://wiki.lyrasis.org/display/DSDOC7x/DSpace+Item+State+Definitions

So, this PR follows that same approach.. it's not very "generic", but it seems defined based on how we define an "embargo" works in DSpace.